packages/blade/src/components/Dropdown/DropdownButton.tsx

```diff
@@ -42,46 +46,48 @@ const _DropdownButton = ({
   return (
     // Using BaseButton here to avoid exporting onBlur and onKeyDown from Button
     // If in future we decide to export onBlur and onKeyDown on Button, this can be replaced with Button
```

Comment:
Why in future, should we expose them? Seems fine to expose really.

Reply:
Oh actually I realised we'll also have to expose accessibility props so its not just these handlers. I'll update the comment but we can probably do it later. Need to figure out if we should expose all accessibility props or somehow limited props with limited values, etc.